rpm/Dockerfile

```diff
@@ -7,5 +7,6 @@ RUN rpmdev-setuptree
 
 USER root
 ADD entry.sh /root/
+COPY ./ /root/rpmbuild/SPECS
```

Comment:
This isn't a big deal but a slight pref to doing it without bundling sources into the build container if possible.

Overall it worked great for me and saved a bunch of time. LGTM.